java-frontend/src/test/files/sym/MostSpecificMethod.java

```diff
@@ -0,0 +1,14 @@
+public class A implements I {
+  public A() {
+    foo(String.class); // resolved to I.foo(...)
```

Comment:
Please remove or fix the comments, they describes the wrong behavior which was observed before.